.changes/unreleased/tinymce-TINY-10795-2024-04-19_2.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Improved
+body: Focus remains on the More button when activated in sliding toolbar mode, preventing focus from shifting to the first control in the expanded toolbar row.
```

Comment:
```suggestion
body: Focus now remains on the `More` button when activated in sliding toolbar mode, rather than moving to the first control in the expanded toolbar row.
```